src/index.html

```diff
@@ -8,10 +8,181 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
+
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="http://127.0.0.1:5500/src/index.html"
+      method="post"
+      class="form_basic"
+      autocomplete="off"
+    >
+      <fieldset class="field_inform">
+        <legend class="form_title_information">
+          Personal information
+        </legend>
+        <label class="label_bottom">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            class="surname_input"
+            required
+          >
+        </label>
+        <label class="label_bottom">
+          Name:
+          <input
+            type="text"
+            name="name"
+            class="name_input"
+            required
+          >
+        </label>
+        <label class="label_bottom">
+          How old are You?
+          <input
+            name="age"
+            type="number"
+            min="1" max="100"
+            value="12"
+          >
+        </label>
+        <label class="label_bottom">
+          Full date of birth:
+          <input
+            type="date"
+            name="dateOfBirth"
+            placeholder="дд.мм.рррр"
+          >
+        </label>
+        <label class="label_bottom">
+          I accept the term  of the agreement
+          <input
+            name="agree"
+            type="checkbox"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="field_inform">
+        <legend class="form_title_registration">
```

Comment:
```suggestion
        <legend>
```

you don't have this class, so it can be deleted